src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  
+  background-color: $base-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 9th point